### DISCUSSION TITLE
Support resolutions field in package.json

### DISCUSSION THREAD

**Discussion by iansu** [npm/feedback#47] [CLI]
Yarn supports the `resolutions` field in `package.json` which allows you to override the resolved version of a package. One example of where this is useful and sometimes even required is with GraphQL. If you have multiple different versions of the `graphql` package present in your project it will output an error and refuse to build or run.

Does npm support or plan to support this or something similar?

**Comment by ljharb** [comment]
https://github.com/npm/rfcs/pull/129

**Reply by iansu** [reply]
Thanks! I did some googling before posting this but didn't find that RFC.
